src/OpenSearch.Client/Descriptors.Indices.cs

```diff
@@ -482,6 +482,7 @@ protected IndexTemplateExistsDescriptor(): base()
 	}
 
 	///<summary>Descriptor for TypeExists <para>https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/exists/</para></summary>
+	///<remarks>Deprecated as of OpenSearch 2.0</remarks>
 	public partial class TypeExistsDescriptor : RequestDescriptorBase<TypeExistsDescriptor, TypeExistsRequestParameters, ITypeExistsRequest>, ITypeExistsRequest
```

Comment:
Should we mark the class as deprecated?
https://stackoverflow.com/questions/314505/how-to-mark-a-class-as-deprecated

Reply:
Unfortunately, no. There are few reasons, one of them: it is not deprecated for OpenSearch 1.0.
https://github.com/Bit-Quill/opensearch-net/pull/31#discussion_r913360046